dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/fat/src/com/ibm/ws/jaxws/fat/FeatureLoggingTest.java

```diff
@@ -0,0 +1,144 @@
+import javax.xml.ws.Dispatch;
+import javax.xml.ws.Service;
+import javax.xml.ws.ServiceMode;
+
+import javax.xml.soap.MessageFactory;
+import javax.xml.soap.SOAPBody;
+import javax.xml.soap.SOAPElement;
+import javax.xml.soap.SOAPEnvelope;
+import javax.xml.soap.SOAPException;
+import javax.xml.soap.SOAPMessage;
+import javax.xml.soap.SOAPPart;
+import javax.xml.ws.soap.SOAPBinding;
+import javax.xml.transform.stream.StreamSource;
+import javax.xml.transform.Source;
+
+import org.apache.cxf.binding.Binding;
+import org.apache.cxf.binding.soap.SoapBinding;
+import org.apache.cxf.ext.logging.event.DefaultLogEventMapper;
+import org.apache.cxf.ext.logging.event.LogEvent;
+import org.apache.cxf.message.ExchangeImpl;
+import org.apache.cxf.message.Message;
+import org.apache.cxf.message.MessageImpl;
+import org.apache.cxf.service.model.BindingInfo;
+import org.apache.cxf.service.model.BindingOperationInfo;
+import org.apache.cxf.service.model.OperationInfo;
+import org.apache.cxf.service.model.ServiceInfo;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.ws.jaxws.test.wsr.server.stub.People;
+import com.ibm.ws.jaxws.test.wsr.server.stub.PeopleService;
+
+import componenttest.annotation.Server;
+import componenttest.annotation.SkipForRepeat;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+import junit.framework.Assert;
+
+@RunWith(FATRunner.class)
+@SkipForRepeat(SkipForRepeat.NO_MODIFICATION)
+public class FeatureLoggingTest {
+
+    @Server("FeatureLoggingServer")
+    public static LibertyServer server;
+
+    private static URL WSDL_URL;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        WebArchive app = ShrinkHelper.buildDefaultApp("helloServer", "com.ibm.ws.jaxws.test.wsr.server",
+                                                      "com.ibm.ws.jaxws.test.wsr.server.impl",
+                                                      "com.ibm.ws.jaxws.test.wsr.server.stub",
+                                                      "com.ibm.ws.jaxws.fat.util");
+        ShrinkHelper.exportDropinAppToServer(server, app);
+
+        server.startServer("FeatureLoggingServer.log");
+
+        assertNotNull("Application hello does not appear to have started.", server.waitForStringInLog("CWWKZ0001I:.*helloServer"));
+        WSDL_URL = new URL(new StringBuilder().append("http://").append(server.getHostname()).append(":").append(server.getHttpDefaultPort()).append("/helloServer/PeopleService?wsdl").toString());
+    }
+
+    @AfterClass
+    public static void tearDown() throws Exception {
+        if (server != null && server.isStarted()) {
+            server.stopServer("CWWKW0056W");
+        }
+    }
+
+    @Test
+    public void testSoap() throws Exception {
```

Comment:
Not sure if this test is necessary, since it's testing with CXF APIs outside of the Liberty runtime right?